### PR TITLE
feedbacks

### PR DESCRIPTION
**Pull request by guillim**
# feedback
We set up a new feature : providing feedbacks

### Constraint
We have to follow the only available endpoint on Haystack for this purpose at the moment : /feedback
The bad news is that we cannot record the exact rating

### How it works
On the answers page, you have 5 stars to click on => if you rate below 3 we record "every answer is bad"
On one answer, if the user clicks on a link => we record a good answer

### Advantage
Since we have constrained ourselves to the SQuAD format, it has a nice advantage : it creates training data for our model ! We can re-use this data very easily to fine-tune our models

### CODE REVIEW

**Review by MichaelBitard — APPROVED** [review]
LGTM
